charts/cinder-csi-plugin/values.yaml

```diff
@@ -23,6 +23,11 @@ csi:
       repository: quay.io/k8scsi/csi-resizer
       tag: v0.5.1
       pullPolicy: IfNotPresent
+  livenessprobe:
+    image:
+      epository: quay.io/k8scsi/livenessprobe
```

Comment:
s/epository/repository

Reply:
thanks :) 

we need a CI for charts .. sorry for the typo